.gitignore

```diff
@@ -56,4 +56,5 @@ yarn-error.log*
 # vscode
 .vscode
 .prettierrc
-.history
\ No newline at end of file
+.history
+sample
```

Comment:
no change at all, should keep last line empty as it is

Reply:
This problem has been resolved